components/eam/src/physics/cam/clubb_intr.F90

```diff
@@ -1946,7 +1991,36 @@ subroutine clubb_tend_cam( &
                      tautmsx,      tautmsy,   cam_in%landfrac )
        call t_stopf('compute_tms')
     endif
-
+        !
+    if (use_od_fd) then
+        gwd_ls=.false.
+        gwd_bl=.false.
+        gwd_ss=.false.
+        gwd_fd=use_od_fd
+        dummy_nm=0.0_r8
+        dummy_utgw=0.0_r8
+        dummy_vtgw=0.0_r8
+        dummy_ttgw=0.0_r8
```

Comment:
What's the point of these dummy variables? Are they always going to be zero?

Reply:
Yes. Because we use interface of od_drags, to keep the inputs in same format, we use these dummy variables.